snapshots/output-net6.0/syntax/Main/QuerySyntax.cs

```diff
@@ -35,92 +35,187 @@ void Select()
           var x = from a in sourceA select a.Method();
 //            ^ definition local 0
 //              documentation ```cs\n? x\n```
+//                     ^ definition local 1
+//                       documentation ```cs\n? a\n```
```

Comment:
Is it expected that the type is `?` in the documentation?

Reply:
I will take a closer look, in some cases roslyn gives not so useful information :)

Reply:
I took a look and actually the symbol that `GetDeclaredSymbol` returns does not contain any type information at all. And the question mark comes from the `SymbolDisplayMiscellaneousOptions.IncludeNullableReferenceTypeModifier` option for displaying the symbol.
I think there is not a lot of things we could do at the moment, with the fact that the symbol return by roslyn does not contain any type information.

Reply:
Fair enough. Thank you for looking into this and confirming 🙏🏻